lib/plugins/create/templates/aws-csharp/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+#build handlers
+dotnet restore
+dotnet publish -c release
+
+#install zip
+apt-get -qq update
+apt-get -qq -y install zip
```

Comment:
Wouldn't that be nice, but unfortunately zip isn't installed on the microsoft supplied dotnet docker images like the one I've used `microsoft/dotnet:1.0.1-sdk-projectjson`

```
...
publish: Published to /app/bin/release/netcoreapp1.0/publish
Published 1/1 projects successfully
/app/bin/Release/netcoreapp1.0/publish /app
./build.sh: line 13: zip: command not found
```